src/Components/WebAssembly/Build/src/targets/ServiceWorkerAssetsManifest.targets

```diff
@@ -178,7 +178,7 @@
 
   <Target Name="_OmitServiceWorkerContent"
     Condition="'$(ServiceWorkerAssetsManifest)' != ''"
-    BeforeTargets="AssignTargetPaths">
+    BeforeTargets="AssignTargetPaths;ResolveCurrentProjectStaticWebAssets">
```

Comment:
For anyone wondering, this line fixes https://github.com/dotnet/aspnetcore/issues/20789